bigbluebutton-html5/imports/ui/components/chat/notification/component.jsx

```diff
@@ -113,13 +113,21 @@ class ChatNotification extends Component {
       disableNotify,
       openChats,
       intl,
+      chatIsOpen,
+      currentChatID,
     } = this.props;
 
     const chatsNotify = openChats
+      .filter(({ unreadCounter }) =>
+        unreadCounter > 0)
       .filter(({ id, unreadCounter }) =>
-        unreadCounter > 0 &&
-        unreadCounter !== this.state.notified[id] &&
-        !disableNotify && id !== PUBLIC_KEY)
+        unreadCounter !== this.state.notified[id])
+      .filter(() =>
+        !disableNotify)
+      .filter(({ id }) =>
+        id !== PUBLIC_KEY)
+      .filter(({ id }) =>
+        !chatIsOpen || id !== currentChatID)
```

Comment:
Could you name those conditionals so its easier to read?

Reply:
`const isPrivateChat = id !== PUBLIC_KEY`

Reply:
And you can leave everything in a single filter if you have readable conditionals.